[Build] Fails on windows

**Description:**
Build fails at composer web module `npm run build` execution

```bash
ERROR in ./~/monaco-editor/esm/vs/language/typescript/lib/typescriptServices.js
    Module not found: Error: Can't resolve 'fs' in 'D:\GitHub\Jerad\ballerina-lang\composer\modules\web\node_modules\monaco-editor\esm\vs\language\typescript\lib'
     @ ./~/monaco-editor/esm/vs/language/typescript/lib/typescriptServices.js 560:69-82
     @ ./~/monaco-editor/esm/vs/language/typescript/tsWorker.js
     @ ./~/monaco-editor/esm/vs/language/typescript/ts.worker.js
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! ballerina-composer@0.974.1 build: `cross-env NODE_ENV=production webpack --config webpack.config.js`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the ballerina-composer@0.974.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Jerad\AppData\Roaming\npm-cache\_logs\2018-06-14T09_09_20_570Z-debug.log
[ERROR] Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 2 (Exit value: 2)
    at org.apache.commons.exec.DefaultExecutor.executeInternal (DefaultExecutor.java:404)
    at org.apache.commons.exec.DefaultExecutor.execute (DefaultExecutor.java:166)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:764)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:711)
    at org.codehaus.mojo.exec.ExecMojo.execute (ExecMojo.java:289)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:290)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:194)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Ballerina - Composer - Web 0.974.1-SNAPSHOT ........ FAILURE [09:29 min]
[INFO] Ballerina - Composer integration tests ............. SKIPPED
[INFO] Ballerina - Distribution ........................... SKIPPED
[INFO] Ballerina - Tools Distribution ..................... SKIPPED
[INFO] Ballerina - Observability - Tracing Test Utils ..... SKIPPED
[INFO] Ballerina - Test Integration ....................... SKIPPED
[INFO] Ballerina - Plugin VSCode Server ................... SKIPPED
[INFO] Ballerina - Plugin VSCode .......................... SKIPPED
[INFO] Ballerina - VSCode Plugin - Parent ................. SKIPPED
[INFO] Ballerina - Microbenchmarks 0.974.1-SNAPSHOT ....... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 09:32 min
[INFO] Finished at: 2018-06-14T14:39:21+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.5.0:exec (npm run build (compile)) on project ballerina-composer-web: Command execution failed.: Process exited with an error: 2 (Exit value: 2)
```

**Affected Versions:**
0.974.1-SNAPSHOT

**OS, DB, other environment details and versions:**
Windows 10, CMD

**Suggested Labels (optional):**
[OS/Windows]

--- a/composer/modules/web/webpack.config.js
+++ b/composer/modules/web/webpack.config.js
@@ -333,6 +333,11 @@ const config = [{
             },
 		]
 	},
+	node: {
+		fs: 'empty',
+		module: 'empty',
+		net: 'empty'
+	},
 	plugins: [
 		new webpack.IgnorePlugin(/^((fs)|(path)|(os)|(crypto)|(source-map-support))$/, /vs\/language\/typescript\/lib/),
 		new webpack.ContextReplacementPlugin(

